tests/APIv2/UsersTest.php

```diff
@@ -129,6 +129,37 @@ public function testDeletePhoto() {
         $this->assertStringEndsWith('/applications/dashboard/design/images/defaulticon.png', $user['photoUrl']);
     }
 
+    /**
+     * Test confirm email is successful
+     */
+    public function testConfirmEmailSucceed() {
+        $emailKey = ['emailKey' => 'test123'];
```

Comment:
You can generate an email confirmation code, like the ones Vanilla would create on its own, by using `UserModel::confirmationCode`. I recommend using that to make this test closer to a real scenario.

Reply:
confirmationCode is private, not sure a work around for that

Reply:
My mistake! This is fine, as is, then.